cfme/tests/physical_infrastructure/test_redfish_physical_server_details.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+import pytest
+
+from cfme.physical.provider.redfish import RedfishProvider
+
+pytestmark = [pytest.mark.provider([RedfishProvider], scope="module")]
+
+
+@pytest.fixture(scope="module")
+def physical_server(appliance, provider, setup_provider_modscope):
+    # Get and return the first physical server
+    yield appliance.collections.redfish_physical_servers.all(provider)[0]
+
+
+def test_redfish_physical_server_details_stats(physical_server):
+    """Navigate to the physical server details page and verify that the stats match"""
+    physical_server.validate_stats(ui=True)
```

Comment:
Optional:
If `physical server` support `create`, `edit` and `remove`. I would like to go with `crud` test and in the same test `validate stats` with `UI`. I think it will be a more effective test.

Reply:
Physical servers do not support CRUD as such. They get populated by the physical infrastructure providers as they get discovered and received trhough its API (Redfish in this case). So as far as the users are concerned, they emerge (or disappear) on their own.